x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/full_time_range_selector/full_time_range_selector_service.ts

```diff
@@ -27,8 +27,7 @@ export async function setFullTimeRange(
   query?: QueryDslQueryContainer,
   excludeFrozenData?: boolean
 ): Promise<GetTimeFieldRangeResponse> {
-  const runtimeMappings = indexPattern.getComputedFields()
-    .runtimeFields as estypes.MappingRuntimeFields;
+  const runtimeMappings = indexPattern.getRuntimeMappings() as estypes.MappingRuntimeFields;
```

Comment:
Since `getRuntimeMappings()` is already typed, we can potential remove this type cast here?

Reply:
Nice catch, fixed!